src/docs/MLOS/MLOSUserGuide.adoc

```diff
@@ -4160,14 +4160,17 @@ NOTE: More information about the source of this task can be found https://github
 | `FOOTPRINT`
 | Defines a geojson file with footprints of the query result.
 | String
+| `PRODUCT_TYPE`
+|  Limits the search to a Sentinel product type.
+|  List [GRD, SLC, OCN (for S1) or S2MSI1C S2MSI2A S2MSI2Ap (for S2)] (default=S2MSI1C)
 | `START_DATE`
 | Determines a start date of the query in the format YYYYMMDD.
 | String
 | `END_DATE`
 | Defines an end date of the query in the format YYYYMMDD.
 | String
 | `USE_START_AND_END_DATE`
-|  If True, it uses the date defined in the START_DATE and START_DATE fields, otherwise it uses the dates defined in th .geojson file.
+|  If True, it uses the date defined in the START_DATE and START_DATE fields, otherwise it downloads all scenes that are published in the last 24 hours.
```

Comment:
If True, it uses the date defined in the START_DATE and START_DATE fields, otherwise it downloads all scenes that were published in the last 24 hours.